framework/wazuh/tests/test_agent.py

```diff
@@ -1593,3 +1593,11 @@ def test_unset_single_group_agent_ko(agent_basic_mock, group_exists_mock, get_gr
     with pytest.raises(WazuhException, match=f".* {expected_exc} .*"):
         Agent.unset_single_group_agent(agent_id, group_id, force=force)
 
+
+def test_agent_check_uninstall_permission():
+    """Check that agent_check_uninstall_permission returns the expected msg"""
+
+    result = agent_check_uninstall_permission()
```

Comment:
```suggestion
    result = check_uninstall_permission()
```

Reply:
Fixed in https://github.com/wazuh/wazuh/commit/efb4a691b97c462325ee2b7e9c4faa616a435692.